plugins/test/unit/plugins/importers/yum/test_sync.py

```diff
@@ -1,7 +1,11 @@
 from copy import deepcopy
 from cStringIO import StringIO
 import os
-import unittest
+
+try:
+    import unittest2 as unittest
+except ImportError:
```

Comment:
How long term do you expect this fallback behavior to be? The challenge with the fallback is that if we can't reliably have the great stuff in `unittest2` then it will fail in environment that fallback if any of our code uses the stuff that `unittest2` actually provides. Does that make sense?